### PR TITLE
feat: request focus for image name when pull image form opened

### PR DESCRIPTION
**Pull request by dgolovin**
### What does this PR do?

It adds focus request for image name input field when pull image form is opened.  In connection with other PR it gives nice and smooth forkflow for pulling images.

### Screenshot/screencast of this [PR #3850](https://github.com/containers/podman-desktop/pull/3850)

https://github.com/containers/podman-desktop/assets/620330/c2230259-b84a-40fd-ada1-0f27332a2027

### What issues does this PR fix or reference?

N/A

### How to test this PR?

1. Open pull image form 
2.  Cursor should be in Image name input, so you can start typing image name right away.

### CODE REVIEW

**Review by deboer-tim — APPROVED** [review]
+1 to this behaviour, and tested. 👍🏼

I'm not sure if we should be using `autofocus` and then disabling the resulting warning with `<!-- svelte-ignore a11y-autofocus -->` though, this almost feels like we're avoiding that. :) Just wait and see what second approver says.

**Review by cdrage — APPROVED** [review]
Unsure if we need a test? But LGTM. Was able to test it and it focuses back to it when opened.

**Review by benoitf — APPROVED** [review]
Requires a component test before merge

we could also bind the element and do focus in the onMount but it's like a tasting preference

**Pull request merged**
